demux/demux_lavf.c

```diff
@@ -443,16 +443,16 @@ static int lavf_check_file(demuxer_t *demuxer, enum demux_check check)
                        priv->avif->name, score, avpd.buf_size,
                        forced_format ? " (forced)" : "");
 
-            for (int n = 0; format_hacks[n].ff_name; n++) {
```

Comment:
Could also add the check into the for loop condition instead, to avoid the extra indentation.

Reply:
Thank you for the advice.